### PR TITLE
fix(InlineEdit): fix calc interpolation issue

### PR DESCRIPTION
**Pull request by glapadre**
Contributes to #2283

Carbon tokens inside of calc needed to be interpolated in order to build correctly

#### What did you change?
`_inline-edit.scss`
`styles.test.js.snap`

#### How did you test and verify your work?
Styling still works correctly and after carbon token is interpolated correctly

### CODE REVIEW

File: packages/cloud-cognitive/src/components/InlineEdit/_inline-edit.scss [hunk 1]
```diff
@@ -105,10 +105,8 @@
       overflow: hidden;
       // positions text and placeholder including when showing placeholder
       // NOTE: Using flex does strange things to a caret in content editable
-      width: calc(100% - var(--#{$block-class}--toolbar-width) - $spacing-05);
-      max-width: calc(
-        100% - var(--#{$block-class}--toolbar-width) - $spacing-05
-      );
+      width: calc(100% - var(--#{$block-class}--toolbar-width) - #{$spacing-05});
+      max-width: calc(100% - var(--#{$block-class}--toolbar-width) - #{$spacing-05});
```
Comment (elycheea) [inline packages/cloud-cognitive/src/components/InlineEdit/_inline-edit.scss:109]: Looks like this part is causes the lint error. It should be fixed with the original indentation and spacing though. 🙂

**Review by elycheea — COMMENTED** [review]
Looks good although we need to fix the lint error and make sure to open this change for v11 as well!

File: packages/cloud-cognitive/src/components/InlineEdit/_inline-edit.scss (hunk 1 above)
Comment (glapadre) [reply packages/cloud-cognitive/src/components/InlineEdit/_inline-edit.scss]: Updated spacing :D

**Review by matthewgallo — APPROVED** [review]
👏

**Comment by matthewgallo** [comment]
As discussed, can you open a PR against the `carbon-v11` branch with these same changes as well?

**Comment by glapadre** [comment]
Done.

**Pull request merged**